compose/local/docs/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM python:3.12-slim-bookworm
+ARG PYTHON_IMAGE_VER=3.12
+
+FROM python:${PYTHON_IMAGE_VER}-slim-bookworm
```

Comment:
won't this not be `slim-bookworm` if we change to 3.13? should we make the whole thing a variable?

Reply:
Python has images for 3.9, 3.10, 3.11, 3.12, 3.13, and even a 3.14 alpha, that all run on slim-bookworm.

Reply:
Ah whoops. ok i am confusing python versions with ubuntu/deban